core/templates/components/button-directives/social-buttons.component.html

```diff
@@ -26,6 +26,12 @@
         <span class="oppia-icon-accessibility-label">Twitter</span>
       </a>
     </div>
+    <div class="oppia-github-follow">
+      <a href="https://github.com/oppia/oppia" target="_blank" rel="noopener">
+        <i class="oppia-follow-icons fa-brands fa-github" style="color: #000;background: #fff;"></i>
```

Comment:
Avoid inline style! Feel free to add a new className if needed!

Reply:
I have removed the inline and introduced a new className.